### PR TITLE
Fix Rubocop code offenses

### PR DESCRIPTION
**Pull request by pvcarrera**

### CODE REVIEW

**Comment by kotp** [comment]
Do you agree with these changes, @pvcarrera ?  I am not sure I do.

The only one that I agree with is the one in anagram.rb.

File: exercises/anagram/anagram_test.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def test_multiple_anagrams
     skip
     detector = Anagram.new('allergy')
     anagrams =
-      detector.match %w( gallery ballerina regally clergy largely leading)
+      detector.match %w(gallery ballerina regally clergy largely leading)
```
Comment (kotp) [inline exercises/anagram/anagram_test.rb]: This change is good.

**Comment by Insti** [comment]
@pvcarrera What Rubocop settings are you using? None of these things generate rubocop warnings on my system.

File: exercises/bowling/bowling_test.rb [hunk 2]
```diff
@@ -139,18 +139,20 @@ def test_should_not_allow_rolls_with_negative_pins
     skip
     assert_raises(
       RuntimeError,
-      'Pins must have a value from 0 to 10') do
-        @game.roll(-1)
-      end
+      'Pins must have a value from 0 to 10',
+    ) do
```
Comment (Insti) [inline exercises/bowling/bowling_test.rb]: I'd prefer to see the assert_raises made into one line rather than 4.

Comment (pvcarrera) [inline exercises/bowling/bowling_test.rb]: Yes, I agree

**Comment by pvcarrera** [comment]
@Insti You are right, I was running a different version run version: `0.42.0`. When I run the project version `bundle exec rubocop` I only get:
`lib/alphametics_cases.rb:1:1: C: Missing frozen string literal comment.`
`lib/generator.rb:1:1: C: Missing frozen string literal comment.`

Should we close this PR then? I can open a new one with only the changes related to this version.

**Comment by kotp** [comment]
I am in the process of disabling all cops until they can be discussed.  The one change is good, but very minor.  Please look at #449 and #443 for further discussion as to what is happening.  Looking forward to your input as well.

**Comment by kotp** [comment]
As to opening a new PR, no need to do that, just force push the changes that you would really like to see happen, @pvcarrera then the reasoning and discussion stays intact.

**Comment by pvcarrera** [comment]
Sorry at the end this PR only removes a withe space 😄

**Comment by kotp** [comment]
Definitely worthwhile!

**Pull request merged**